app/models/chargeback_rate_detail.rb

```diff
@@ -175,11 +178,16 @@ def contiguous_tiers?
         error = true if tier.ends_with_infinity?
       end
 
-      break if error
+      ambiguous_error = previous_tiers_with_same_start_and_finish && consecutive_tiers?(tier, tier)
```

Comment:
@lpichler, I tested it and it doesn't work to me :/ I think that the following line should be:
`ambiguous_error = previous_tiers_with_same_start_and_finish || consecutive_tiers?(tier, tier)`
what do you think?